convert_hf_to_gguf.py

```diff
@@ -525,6 +525,11 @@ def get_vocab_base(self) -> tuple[list[str], list[int], str]:
             else:
                 token: str = reverse_vocab[i]
                 if token in added_vocab:
+                    # We need to manually encode and decode the added tokens in case special characters
+                    # used for `\n` / `\t` have been manually added in the added tokens
+                    if len(token) == 1:
```

Comment:
```suggestion
                    # To avoid unexpected issues - we make sure to encode single-char tokens
                    if len(token) == 1:
```

Reply:
I'm looking at the Falcon tokenizer and I don't see any added tokens that have `\n` or `\t`: https://huggingface.co/tiiuae/Falcon3-7B-Instruct/raw/main/tokenizer.json

For which tokens does this change make a difference?

Maybe also add some logs to know when this path is being triggered so we can spot any potential problems with other models.

Reply:
Chiming in here ! The added token is 
```
    {
      "id": 12,
      "content": "Ċ",
      "single_word": false,
      "lstrip": false,
      "rstrip": false,
      "normalized": false,
      "special": false
    }
``` (`\t` is the id 13)
the only way to convert it properly to `\n` is to encode / decode using the tokenizer

Reply:
I just added a log message inside the if statement.